src/client/app/containers/CompareChartContainer.ts

```diff
@@ -84,6 +86,14 @@ function mapStateToProps(state: RootState, ownProps: CompareChartContainerProps)
 		}
 	}
 
+	// Display Plotly Buttons Feature
+	// The number of items in defaultButtons and advancedButtons must differ as discussed below
+	const defaultButtons: Plotly.ModeBarDefaultButtons[] = ['zoom2d', 'pan2d', 'select2d', 'lasso2d', 'zoomIn2d', 'zoomOut2d', 'autoScale2d',
+		'resetScale2d'];
+	const advancedButtons: Plotly.ModeBarDefaultButtons[] = ['select2d', 'lasso2d', 'autoScale2d', 'resetScale2d'];
+	// Manage button states with useState
+	const	[listOfButtons, setListOfButtons] = React.useState(defaultButtons);
```

Comment:
When I click this icon it crashes OED. The error relates to using a Hook (useState, I think) outside a component. This does not use a component as the other graphics do as it is a container. It either needs a modified solution or the component needs to be converted. I doubt you want to do the later so either find a fix or remove it for now from this graphic and leave a message that it needs to be done.